content/reference/plugins/github_actions/includes/use-case-build-and-run.gradle.md

```diff
@@ -0,0 +1,52 @@
+```yaml
```

Comment:
I don't think we have this mechanism (download of the sample file) in the doc system at the moment. Not sure it would be that useful in this specific case compared to the "copy to clipboard" button? 🤷